### ISSUE TITLE
Can't download one e-book from the Kindle app

### ISSUE THREAD

**Issue by inferno986return** [noDRM/DeDRM_tools#286]
### Question / bug report

![image](https://user-images.githubusercontent.com/1838420/218819758-42fbcacb-f694-4fdd-8672-9590c279560d.png)

This book was released today and I speculate that either Amazon or the authors have forced the new DRM which this version of Kindle for PC can't handle. The latest Kindle for PC release downloads this book no problem in the KFX-ZIP format though expectedly the tool can't open those.

### Which version of Calibre are you running?

6.10

### Which version of the DeDRM plugin are you running?

10.0.3

### If applicable, which version of the Kindle software are you running?

1.26

### Log output

```log
Paste log output here.
```

**Comment by ElleKayEm** [comment]
Yes, books published after 3 Jan, 2023 can't be downloaded with any of the previous Kindle for PC/Mac apps.  Which, as you see, means they are the latest KFX format which has DRM that can't be removed.  If you have an E-ink Kindle, you can use the download & transfer option from the website with purchased books to get the older format which can be decrypted.  See this post for all known details about this:  https://www.mobileread.com/forums/showthread.php?t=283371

More discussion here: https://www.mobileread.com/forums/showthread.php?t=351285
